anatomy

```diff
@@ -2364,9 +2364,10 @@ an overview of the purpose of the sections:
 || 1 || Commands ||
 || 2 || System Calls ||
 || 3 || C Functions and Libraries ||
-|| 4 || File formats ||
-|| 5 || Miscellany, including Standards, Macros, Environments, etc. ||
-|| 7 || Special Files ||
+|| 4 || Special Files ||
+|| 5 || File formats ||
+|| 7 || Miscellany, including Standards, Macros, Environments, etc. ||
+|| 8 || Operator Commands ||
```

Comment:
There's a bit of a disconnect between the descriptions offered here and what I'm seeing in Rich's `msec.in`.

https://github.com/richlowe/illumos-gate/blob/8659cae58e129cc50d463219993c7d2fd0f9303e/usr/src/cmd/mandoc/msec.in

Reply:
These are slightly different because I'm describing the entire section as opposed to that which has more detailed subsections all broken out.

Reply:
I was referring to the top-level section titles being different. E.g.

```
LINE("4",		"Device and Network Interfaces")
LINE("5",		"File Formats and Configurations")
LINE("7",		"Standards, Environments, and Macros")
LINE("8",		"Maintenance Commands and Procedures")
```

Reply:
Yeah, I got what you were trying to say. I was trying to say that the actual contents of the sections don't actually match those per se which is wy I had a more general/different statement. I guess it sounds like not matching 100% is bad so I should change that yet again. Our use of chapters makes this different. For example, part of the problem is that our section 4 contains file systems, I don't know what 'procedures' refers to, etc. But it sounds like being different is worse.

Reply:
I understood what you meant about subsections. I just found it odd that the section titles we present in the book are different from those we ship in the running system.

> part of the problem is that our section 4 contains file systems

I agree that it's weird that the section title presented by the system is "Device and Network Interfaces" when it includes other things. Perhaps `msec.in` should be changed? Perhaps neither section title is good and there's a better, third option?

> I don't know what 'procedures' refers to

I don't either. I thought all pages in 1M (now 8) described binary executables.

Perhaps what I really should be doing is filing an issue with the IPD4-related work and not this. My intent with the original comment was to point out the two don't agree when I would have expected they would.

I'm fine with this staying as-is. I do not want to over rotate on this and create more work for you.

Reply:
It's fine. I'm going to change to match the current things there and then if we want changes change it after that's all integrated.